server/aws-lsp-identity/src/language-server/profiles/profileService.ts

```diff
@@ -0,0 +1,245 @@
+import {
+    ListProfilesError,
+    ListProfilesParams,
+    ListProfilesResult,
+    Profile,
+    ProfileKind,
+    SsoSession,
+} from '@aws/language-server-runtimes/protocol/identity-management'
```

Comment:
nit: for Servers implemented in `language-servers` repo recommendation is to import types from `@aws/language-server-runtimes/server-interface` directory instead of `protocol`. `server-interface` is setup to re-export most of `protocol` where it make sense, so all there types must be available.

Since Servers implementations are decoupled from LSP Clients using `/server-interface`, this provides better separation of concerns between what Client and Server should implement. `/protocol` is intended to be used by  Client extensions implementors.

Reply:
I ran into this problem last night too.  identity-management wasn't exported with server-interface, which is why they were imported from protocol.  Please check this PR https://github.com/aws/language-server-runtimes/pull/219 to add identity-management to server-interface then I will update these imports here.